tools/backups2datalad/datasetter.py

```diff
@@ -204,9 +221,12 @@ def get_dandisets(
             else:
                 yield d
 
-    def get_dandiset_stats(self, ds: Dataset) -> DandisetStats:
+    def get_dandiset_stats(
+        self, ds: Dataset
+    ) -> Tuple[DandisetStats, Dict[str, DandisetStats]]:
         files = 0
         size = 0
+        substats: Dict[str, DandisetStats] = {}
```

Comment:
I see now why "sub"stats, I guess ok to keep it sub here but assumption outside is that they do belong to zarr subdatasets... we might need to RF if we gain some other subdatasets at some point